### PR TITLE
feat: added Awesome Tauri to community resources

### PR DESCRIPTION
**Pull request by simonhyll**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)
- New or updated content
- Translated content
- Changes to the docs site code
- Something else!

#### Description

- Closes # 
- What does this PR change? Give us a brief description. 
- Did you change something visual? A before/after screenshot can be helpful.

### CODE REVIEW

File: src/content/docs/features/index.mdx [hunk 1]
```diff
@@ -17,6 +18,10 @@ Tauri comes with extensibility in mind. On this page you'll find:
 	<FeaturesList />
 	## Community Resources
 	<CommunityList />
+	## Plugins
+	<AwesomeTauri section="integrations" />
+	## Integrations
+	<AwesomeTauri section="integrations" />
```
Comment (vasfvitor) [inline src/content/docs/features/index.mdx]: repeated sections
```suggestion
	<AwesomeTauri section="plugins" />
	## Integrations
	<AwesomeTauri section="integrations" />
```

Comment (vasfvitor) [reply src/content/docs/features/index.mdx:24]: and the title

Comment (simonhyll) [reply src/content/docs/features/index.mdx]: Fixed

**Review by vasfvitor — COMMENTED** [review]
oddly the "Nothing found" doesn't appears sometimes, I know those are supposedly to be removed as you did in your website. So I think it's better to first alter the search feature then see how it behaves here. What do you think?

below "plugins":

below "integrations":


A note about the search feature: I kept trying to use the website search instead of rolling up to the search component, did it ever happen to you?

**Comment by simonhyll** [comment]
@vasfvitor Haven't really had issues with using the wrong search field no, but I can see why people might

Something is definitely wrong with how I'm applying the hidden class 🤔

**Comment by vasfvitor** [comment]
> Something is definitely wrong with how I'm applying the hidden class 🤔

yeah, but once  #1900 is in I think this wouldn't be a problem anymore

**Pull request merged**